app/controllers/manuals_controller.rb

```diff
@@ -34,47 +39,11 @@ def updates
 private
 
   def presented_manual
-    @presented_manual ||=
-      ManualPresenter.new(manual: manual, view_context: view_context)
-  end
-
-  def ensure_manual_is_found
-    if content_store_manual.nil?
```

Comment:
Do we still need to keep the `.nil?` test?

Reply:
I don't think so, see the comment below.